python/base_computer/upgrade_view.py

```diff
@@ -150,7 +150,27 @@ def process_resource(key, resource_text):
     original = float(parts[original_index])
 
     return f"{format_number(damaged)} (orig: {format_number(original)})"
+
+def process_range(text):
+    parts = text.split('/')
+
+    damaged_index = 0
+    original_index = 0
+
+    if len(parts) == 2:
+        damaged_index = 0
+        original_index = 1
+    elif len(parts) == 3:
+        damaged_index = 1
+        original_index = 2
+    else:
+        return text
```

Comment:
do we want to log the last case to help with debugging? Or would it always be valid?
Some comments on the above would be helpful to understand the 3 cases.

Reply:
Added documentation.